easybuild/easyblocks/s/suitesparse.py

```diff
@@ -172,8 +184,25 @@ def configure_step(self):
 
         else:
             # after v6.0.0, no option for metis, its own metis is used anyway
-            # nothing to do here, set the CMAKE_OPTIONS in easyconfigs
-            pass
+            # set CMAKE_OPTIONS if it is not specified in easyconfigs
+            # CMAKE_INSTALL_PREFIX is managed by easybuild
+            cmake_options = '-DCMAKE_INSTALL_PREFIX=%s' % self.installdir
+
+            flexiblas = get_software_root('FlexiBLAS')
+            mkl = get_software_root('imkl')
+            openblas = get_software_root('OpenBLAS')
+
+            if flexiblas:
+                cmake_options = ' '.join([cmake_options, '-DBLA_VENDOR=FlexiBLAS'])
+            elif mkl:
+                cmake_options = ' '.join([cmake_options, '-DBLA_VENDOR=Intel'])
+            elif openblas:
+                cmake_options = ' '.join([cmake_options, '-DBLA_VENDOR=OpenBLAS'])
+            else:
+                raise EasyBuildError("No FlexiBLAS/MKL/OpenBLAS is founded. "
+                                     "Please assign BLA_VENDOR in cmake_options in easyconfigs")
```

Comment:
As it is now, setting `BLA_VENDOR` in `cmake_options` won't help to avoid this error, since we're not checking whether `BLA_VENDOR` is defined in `cmake_options`?

Reply:
Right. I did not check it carefully. Now I have fixed it. Thank you!